packages/ckeditor5-dev-env/tests/release-tools/changelog/transformcommit.js

```diff
@@ -121,5 +121,73 @@ describe( 'dev-env/release-tools/changelog/writer-options', () => {
 			expect( stubs.logger.info.calledOnce ).to.equal( true );
 			expect( stubs.logger.info.firstCall.args[ 0 ] ).to.match( /\* 684997d "Invalid commit\." \u001b\[31mINVALID/ );
 		} );
+
+		it( 'does not duplicate the commit header in additional description for merge commits', () => {
+			const commitDescription = [
+				'* Added interactive mode for generating the changelog. A user can provide a new version or skip the process for given package.',
+				'* Release task - rebuilt module for collecting dependencies to release.',
+				'* Release task - a new version to release will be read from CHANGELOG file.',
+				'* Changelog task - will not break if changelog does not exist.',
+				'* Used `semver` package for bumping the version (instead of a custom module).',
+			];
+			const commitDescriptionWithIndents = commitDescription.map( ( l ) => `   ${ l }` ).join( '\n' );
```

Comment:
Tests must be easy to read. They should be as straightforward as possible. I've been looking for a test which checks whether the entries are being indented and couldn't find it because first you see here is non-indented list items. A good test should be a no brainer. Easy to analyze and easy to find.